database/migrations/2018_05_12_075944_create_transactions_table.php

```diff
@@ -21,6 +21,7 @@ public function up()
             $table->string('description');
             $table->unsignedInteger('category_id')->nullable();
             $table->unsignedInteger('partner_id')->nullable();
+            $table->unsignedInteger('loan_id')->nullable();
```

Comment:
This commit contains changes on `tables` table, for existing database, run this SQL command :

```sql
ALTER TABLE `transactions`
	ADD COLUMN `loan_id` INT(10) UNSIGNED NULL DEFAULT NULL AFTER `partner_id`;
```

Please be sure to backup your database before run the command. Thanks.